src/database/mongo_database.rs

```diff
@@ -1,12 +1,13 @@
+use mongodb::error::Error;
 use mongodb::options::ClientOptions;
 use mongodb::Client;
-use mongodb::error::Error;
+use log::info;
 
 pub async fn connect() -> Result<Client, Error> {
     info!("Initializing database connection");
 
-    let mut client_options = ClientOptions::parse("mongodb://localhost:27017").await?;
+    let mut client_options = ClientOptions::parse("").await?;
     client_options.app_name = Some("Actix Web Server".to_string());
```

Comment:
Is the address intended to be empty?  Also, we should use something more descriptive than `"Actix Web Server"` for the client name.